themes/OpenFisca-2022/layouts/partials/flag.html

```diff
@@ -0,0 +1 @@
+<div class="flag flag-{{ slicestr .jurisdiction 0 2 }} {{ with .class }}{{ . }}{{ end }}"><span>{{ .jurisdiction }}</span></div>
```

Comment:
Display country code on hover to help with identifying flags 🙂 
```suggestion
<div class="flag flag-{{ slicestr .jurisdiction 0 2 }} {{ with .class }}{{ . }}{{ end }}" title="{{ .jurisdiction }}"><span>{{ .jurisdiction }}</span></div>
```